_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -14,6 +16,26 @@ class AnnualIncomeSection extends React.Component {
   }
 
   // TODO: Figure out best way to enable users to change their response to pension
+
+  updateChildren() {
+    if (this.props.external === undefined || this.props.external.children === undefined) return;
```

Comment:
Do you need this check? `this.props.external.children` should never be `undefined`, even if there are no children because it is initialized to an empty array. You're passing the children to this section if the path is `annual-income`, which it would always be for this section. We probably want to write tests to verify that `this.props.external.children` is never undefined instead of checking for it here. If there's ever a case where it could be undefined, that probably means something is wrong with how we've structured the section and `getExternalData`.